### DISCUSSION TITLE
スマートフォンからだと重くて開けない記事に関して

### DISCUSSION THREAD

**Discussion by eduidl** [increments/qiita-discussions#287] [要望・アイデア・不具合]
文字数が多かったり、数式が多く使われているなどして重い記事が存在しており、スマートフォンから開こうとするとブラウザごと落ちることがあります。
ユーザが自由に記事を書く都合上、ページング等の対策が難しく、このようになっているのだと思いますが、改善を検討いただけないでしょうか？

### 補足

- 機種：Pixel 4a
- ブラウザ：Vivaldi

**Comment by tanaka-asami415** [comment]
@eduidl 
ご意見いただきありがとうございます。
パフォーマンス改善には努めておりますが、ご不便をおかけして申し訳ございません。
もしよろしければ、開けなかったページを教えていただけると幸いです🙇

**Reply by eduidl** [reply]
直近開いて覚えている限りだと https://qiita.com/takobaya391076/items/49b15c1fa36734b3fa53 です。

**Reply by tanaka-asami415** [reply]
@eduidl 
ありがとうございます🙇
コードブロック等のパフォーマンスに改善が必要なことがわかりました。
パフォーマンス改善の一環として対応を検討いたします。
